allowed-meta.d.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * List of meta keys permitted for clients.
+ *
+ *```js
+ * const { ALLOWED_META } = require('@logux/server')
+ * async function outMap (action, meta) {
+ *   const filtered = { }
+ *   for (const i in meta) {
+ *     if (ALLOWED_META.includes(i)) {
+ *       filtered[i] = meta[i]
+ *     }
+ *   }
+ *   return [action, filtered]
+ * }
+ * ```
+ */
+export const ALLOWED_META: readonly ['id', 'time', 'subprotocol']
```

Comment:
Why do need to specify exact value in types?

Reply:
It's not obligatory. But more accurate types are more convenient.
We can replace with
```ts
export const ALLOWED_META: string[];
```

Reply:
I afraid that I will forget to update types after `ALLOWED_META` update